packages/components/src/test/properties/inputs/UniquePropertyValuesSelector.test.tsx

```diff
@@ -478,6 +481,250 @@ describe("UniquePropertyValuesSelector", () => {
     });
   });
 
+  it("does not try to fetch another page on error", async () => {
+    getDistinctValuesIteratorStub.rejects();
+
+    const { getByText, user } = render(
+      <TestComponentWithPortalTarget>
+        <UniquePropertyValuesSelector property={propertyDescription} onChange={() => {}} imodel={testImodel} descriptor={descriptor} />,
+      </TestComponentWithPortalTarget>,
+    );
+
+    // open menu
+    const menuSelector = await waitFor(() => getByText("unique-values-property-editor.select-values"));
+    await user.click(menuSelector);
+
+    // add some delay to see if a second page load will be requested
+    await new Promise((f) => setTimeout(f, 1000));
```

Comment:
This looks hacky. Can we remove this?

Reply:
I tried to avoid it when writing the test, however I was not able to since we need to wait for `AsyncSelect` to try to load the second page (which might not happen) so there is no element that we could wait for.

Reply:
Then maybe we don't need this test and it's enough to validate it manually?

Reply:
Removed the timeout part and kept the renamed test for coverage.